### PR TITLE
Hangman Game in C++

### PR DESCRIPTION
**Pull request by Chandra0804**
#### Checklist

- [x ] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md#New-File-Name-guidelines)
- [x] Relevant documentation/comments is changed or added
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

File: others/Hangman_game/Hangman.cpp [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#include <iostream>
+#include <string.h>
```
Comment (Panquesito7) [inline others/Hangman_game/Hangman.cpp:2]:
```suggestion
#include <cstring>
```

File: others/Hangman_game/Hangman.cpp [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+#include <fstream>
+#include <time.h>
```
Comment (Panquesito7) [inline others/Hangman_game/Hangman.cpp:4]:
```suggestion
#include <ctime>
```

File: others/Hangman_game/Hangman.cpp [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+using namespace std;
```
Comment (Panquesito7) [inline others/Hangman_game/Hangman.cpp:5]: This is discouraged. Please strictly use `std::` for each of its members.

File: others/Hangman_game/Hangman.cpp [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+void print_string(char b[], int k, int len)
```
Comment (Panquesito7) [inline others/Hangman_game/Hangman.cpp:6]: Functions should be documented explaining the use of its parameters, a brief description, and what it returns.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Feel free to add the game in a new folder named `games`. 🙂

File: DIRECTORY.md [hunk 5]
```diff
@@ -267,6 +267,8 @@
   * [Union Of Two Arrays](https://github.com/TheAlgorithms/C-Plus-Plus/blob/HEAD/operations_on_datastructures/union_of_two_arrays.cpp)
 
 ## Others
+  * Hangman Game
```
Comment (krishnacx) [inline DIRECTORY.md:270]: This is not required

File: DIRECTORY.md [hunk 6]
```diff
@@ -267,6 +267,8 @@
...
+    * [Hangman](https://github.com/TheAlgorithms/C-Plus-Plus/blob/HEAD/others/Hangman_game/Hangman.cpp)
```
Comment (krishnacx) [inline DIRECTORY.md:271]: Follow the alphabetical order

File: others/Hangman_game/Hangman.cpp (hunk 2 above)
Comment (krishnacx) [inline others/Hangman_game/Hangman.cpp:4]: Add header comments

**Pull request closed without merging**